examples/tools/cmake/cmake_toolchain/use_different_toolchain_generator.rst

```diff
@@ -0,0 +1,104 @@
+
+.. _examples-tools-use-different-toolchain-generator:
+
+Using CMakeToolchain with Different Generators: Ninja Example
+=============================================================
+
+This guide demonstrates how to use ``CMakeToolchain`` with predefined generators like ``Ninja`` and how to configure it to use different generators.
+
+Creating a Basic Project
+-------------------------
+We will create a basic project based on the ``cmake_exe`` template as an example of a C++ project:
+
+.. code-block:: bash
+
+    $ conan new -d name=foo -d version=0.1.0 cmake_exe
+
+Understanding CMake Generators
+------------------------------
+CMake offers a variety of `generators <https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html>`_ to create build system files. If you want to use a generator other than the default chosen by CMake, you can configure ``tools.cmake.cmaketoolchain:generator``.
```

Comment:
I'd add a note here to clarify the user not to get consued with Conan's generators (`CMakeToolchain` etc), and CMake's (`Ninja` etc)